nixos/modules/services/web-servers/apache-httpd/mediawiki.nix

```diff
@@ -91,12 +91,16 @@ let
     };
 
     skins = config.skins;
+    extensions = config.extensions;
 
     buildPhase =
       ''
         for skin in $skins; do
           cp -prvd $skin/* skins/
         done
+        for extension in $extensions; do
+          cp -prvd $extension/* extensions/
```

Comment:
> I'm not familiar with mediawiki

Well I presume you are testing this somehow? So if you can try and see if it works. I don't see why it wouldn't unless mediawiki was doing something quite odd.

> It might try to write to them.

This won't be allowed in either case as the nix store is mounted read only.

> As to the /*, note that $extension isn't a single extension; it's a directory containing extensions. ...

Ah, I see. That's fine then.